noir-projects/noir-contracts/contracts/test_contract/Nargo.toml

```diff
@@ -6,6 +6,7 @@ type = "contract"
 
 [dependencies]
 aztec = { path = "../../../aztec-nr/aztec" }
+address_note = { path = "../../../aztec-nr/address-note" }
```

Comment:
Importing this here now instead of the local TestNote which I nuked because it was just a very weird thing at this point. Wanted to retain the same num of notes in the contract to test the note type id derivation in `test.nr`.